project_purchase_link/i18n/fr.po

```diff
@@ -0,0 +1,71 @@
+# Translation of Odoo Server.
```

Comment:
@acsonefho The good path for translations is weblate: https://translation.odoo-community.org/